### PR TITLE
Rewrite host

### PR DESCRIPTION
**Pull request by rcspringer**
Please check the code

### CODE REVIEW

File: Modules/Products.php [hunk 1]
```diff
@@ -1,20 +1,21 @@
...
-        $sth->bindParam(1,$id);
-        $sth->execute();
-        return $sth->fetchAll(PDO::FETCH_CLASS,'Product')[0];
-    }
\ No newline at end of file
+class Product{}
+
+function getProducts(int $categoryId)
+{
+    global $pdo, $params;
```
Comment (gijsdl) [inline Modules/Products.php:7]: Remove unused variable

File: Templates/categories.php [hunk 2]
```diff
@@ -1,47 +1,43 @@
...
-        <link rel="stylesheet" href='/healthone/public/css/bootstrap.min.css' />
-        <script src='/healthone/public/js/jquery-3.2.1.min.js'></script>
-        <script src='/healthone/public/js/bootstrap.min.js'></script>
-    </head>
+<?php
+include_once('defaults/head.php');
+?>
 
-    <body style="background-color: gainsboro">
+<body style="background-color: gainsboro">
```
Comment (gijsdl) [inline Templates/categories.php:7]: Style inline to CSS

File: Templates/categories.php [hunk 3]
```diff
@@ -1,47 +1,43 @@
...
-                <div class="col-sm-4 col-md-3">
-                    <div class="panel panel-primary">
-                        <div class="panel-heading"><?=$category->name?></div>
-                        <a href="products/<?=$category->id?>">
-                            <div class="panel-body">
-                                <img class="img-responsive center-block" style="width:70%" src='/healthone/public/img/<?= $category->picture ?>' />
-                            </div>
-                        </a>
-                    </div>
+<div class="container" style="background-color: white">
```
Comment (gijsdl) [inline Templates/categories.php:9]: Style inline to CSS

File: Templates/categories.php [hunk 4]
```diff
@@ -1,47 +1,43 @@
...
+    <h4>Categorieën</h4>
+    <div class="row">
+        <?php global $categories ?>
+        <?php foreach ($categories as $category): ?>
+            <div class="col-sm-4 col-md-3">
+                <div class="panel panel-primary">
+                    <div class="panel-heading"><?= $category->name ?></div>
+                    <a href="/categories/<?= $category->id ?>">
+                        <div class="panel-body">
+                            <img class="img-responsive center-block" style="width:70%"
```
Comment (gijsdl) [inline Templates/categories.php:24]: Style inline to CSS

File: Templates/defaults/footer.php [hunk 5]
```diff
@@ -1,8 +1,6 @@
 <footer>
-
-    <img class="pull-right img-thumbnail" width="40"  style="margin-top: -10px;margin-bottom: 10px" src='/healthone/public/img/linkedin.jpg' />
-    <img class="pull-right img-thumbnail" width="40"  style="margin-top: -10px;margin-bottom: 10px" src='/healthone/public/img/tweet.jpg' />
-    <img class="pull-right img-thumbnail " width="40"  style="margin-top: -10px;margin-bottom: 10px" src='/healthone/public/img/pinterest.png' />
-
+    <img class="pull-right img-thumbnail" width="40"  style="margin-top: -10px;margin-bottom: 10px" src="/img/linkedin.jpg" />
+    <img class="pull-right img-thumbnail" width="40"  style="margin-top: -10px;margin-bottom: 10px" src="/img/tweet.jpg" />
+    <img class="pull-right img-thumbnail " width="40"  style="margin-top: -10px;margin-bottom: 10px" src="/img/pinterest.png" />
```
Comment (gijsdl) [inline Templates/defaults/footer.php:4]: Style inline to CSS

File: Templates/defaults/header.php [hunk 6]
```diff
@@ -1,10 +1,7 @@
-
 <header>
-       <div class="row" style="margin-top: 20px">
-
-           <div class="col-sm-11">
-               <h1 >HealthOne <span class="text-danger">Sportcentrum</span></h1>
-           </div>
-
-       </div>
+    <div class="row" style="margin-top: 20px">
```
Comment (gijsdl) [inline Templates/defaults/header.php:2]: Style inline to CSS

File: Templates/defaults/pictures.php [hunk 7]
```diff
@@ -1,5 +1,5 @@
 <figure>
-    <img style="margin-bottom: 20px" class="img-responsive" src='/healthone/public/img/healthone-wide.png' />
+    <img style="margin-bottom: 20px" class="img-responsive" src='/img/healthone-wide.png' />
```
Comment (gijsdl) [inline Templates/defaults/pictures.php:2]: Style inline to CSS

File: Templates/home.php [hunk 8]
```diff
@@ -1,38 +1,26 @@
...
-    include_once ('defaults/footer.php');
-
+    include_once('defaults/head.php');
     ?>
-</div>
-
-
-</body>
+    <body style="background-color: gainsboro">
+        <div class="container" style="background-color: white">
```
Comment (gijsdl) [inline Templates/home.php:7]: Style inline to CSS

File: Templates/product.php [hunk 9]
```diff
@@ -1,47 +1,35 @@
...
-    <title>HealthOne!</title>
-    <link rel="stylesheet" href='/healthone/public/css/bootstrap.min.css' />
-    <script src='/healthone/public/js/jquery-3.2.1.min.js'></script>
-    <script src='/healthone/public/js/bootstrap.min.js'></script>
-</head>
+<?php
+include_once('defaults/head.php');
+?>
 
 <body style="background-color: gainsboro">
```
Comment (gijsdl) [inline Templates/product.php:7]: Style inline to CSS

File: Templates/product.php [hunk 10]
```diff
@@ -1,47 +1,35 @@
...
 
 <div class="container" style="background-color: white">
```
Comment (gijsdl) [inline Templates/product.php:9]: Style inline to CSS

File: Templates/product.php [hunk 11]
```diff
@@ -1,47 +1,35 @@
...
-
-                </div>
-
-
-
+        <div class="panel panel-primary">
+            <div class="panel-heading"><?= $product->name ?></div>
+
+            <div class="panel-body">
+                <img class="img-responsive center-block" style="width:70%" src='/img/<?= $product->picture ?>'/>
```
Comment (gijsdl) [inline Templates/product.php:24]: Style inline to CSS

File: Templates/products.php [hunk 12]
```diff
@@ -1,13 +1,8 @@
...
-    <title>HealthOne!</title>
-    <link rel="stylesheet" href='/healthone/public/css/bootstrap.min.css' />
-    <script src='/healthone/public/js/jquery-3.2.1.min.js'></script>
-    <script src='/healthone/public/js/bootstrap.min.js'></script>
-</head>
+<?php
+include_once('defaults/head.php');
+?>
 
 <body style="background-color: gainsboro">
```
Comment (gijsdl) [inline Templates/products.php:7]: Style inline to CSS

File: Templates/products.php [hunk 13]
```diff
@@ -16,17 +11,17 @@
...
     <div class="row">
         <?php foreach ($products as $product): ?>
             <div class="col-sm-4 col-md-3 col-lg-2">
                 <div class="panel panel-primary">
                     <div class="panel-heading"><?=$product->name?></div>
-                    <a href="/healthone/product/<?=strtolower($name)?>/<?=$product->id?>">
+                    <a href="/categories/<?=$categoryId?>/product/<?=$product->id?>">
                         <div class="panel-body">
-                            <img class="img-responsive center-block" style="width:70%" src='/healthone/public/img/<?= $product->picture ?>' />
+                            <img class="img-responsive center-block" style="width:70%" src='/img/<?= $product->picture ?>' />
```
Comment (gijsdl) [inline Templates/products.php:24]: Style inline to CSS

**Review by gijsdl — CHANGES_REQUESTED** [review]

File: Modules/Products.php (hunk 1 above)
Comment (rcspringer) [reply Modules/Products.php]: Should be fixed

File: Templates/categories.php (hunk 2 above)
Comment (rcspringer) [reply Templates/categories.php]: Fixed

File: Templates/categories.php (hunk 3 above)
Comment (rcspringer) [reply Templates/categories.php]: Fixed

File: Templates/categories.php (hunk 4 above)
Comment (rcspringer) [reply Templates/categories.php]: Fixed

File: Templates/defaults/footer.php (hunk 5 above)
Comment (rcspringer) [reply Templates/defaults/footer.php]: Fixed

File: Templates/product.php (hunk 9 above)
Comment (rcspringer) [reply Templates/product.php]: Fixed

File: Templates/products.php (hunk 13 above)
Comment (rcspringer) [reply Templates/products.php]: Fixed

**Review by gijsdl — APPROVED** [review]

**Review by mjlinden — APPROVED** [review]
<base href="http://healthone/" target="_blank">
Wanneer de url -> healthone/products/1
dan worden resources gezocht in healthone/products/img etc
deze regel in de header zorgt ervoor dat resources nu gezocht worden in
healthone/img/ etc

**Pull request merged**